model/commitq/githubpr/github_pr_sender.go

```diff
@@ -0,0 +1,91 @@
+package githubpr
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/google/go-github/github"
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+)
+
+type githubPRLogger struct {
+	prService    *github.PullRequestsService
+	statusSender send.Sender
+	*send.Base
+}
+
+// NewGithubPRLogger creates a new Sender implementation that
+// merges a pull request
+// Specify an OAuth token for GitHub authentication
+func NewGithubPRLogger(name string, token string, statusSender send.Sender) (send.Sender, error) {
+	ctx := context.TODO()
```

Comment:
`context.TODO()` means something like "I'm going to change this function to accept a context but haven't done it yet. If that's true, then this is fine. But I think you might actually want to construct a new context with `context.Background()`, and maybe even call `WithTimeout` and `defer` the `CancelFunc`.

Reply:
Here changed to context.Background() (pending discussion with Sam about stashing contexts)

Reply:
This context no longer is used.